app/containers/Login/constants.js

```diff
@@ -0,0 +1,3 @@
+export const LOGIN_ACTION = 'CALL USER LOGIN';
```

Comment:
ricordati di tenere le costanti in sync col loro contenuto: LOGIN_ACTION = 'LOGIN_ACTION';
se no rischi di perderti quando fai magari debug con l'estensione di redux o stampando qualche console log in giro.